openmc/region.py

```diff
@@ -231,6 +235,16 @@ def __str__(self):
     def nodes(self):
         return self._nodes
 
+    @property
+    def bounding_box(self):
+        ll = np.array([-np.inf, -np.inf, -np.inf])
```

Comment:
I'm personally a fan of the more verbose `lower_left`, `upper_right` notation compared to `ll` and ``ur`, especially without a docstring or other comment explaining what is going on here (as is the case for the `Surface.bounding_box` properties. But this is certainly a minor point so it's up to you.